analytics.js

```diff
@@ -853,7 +853,14 @@ class Analytics {
             }
           }
         } catch (err) {
-          err.message = `[sendToNative]:: [Destination: ${obj.name}]:: "${err.message}"`;
+          if (err instanceof Error) {
+            err.message = `[sendToNative]:: [Destination: ${obj.name}]:: "${err.message}"`;
+          } else if (typeof err === 'string') {
```

Comment:
Are there any places in the SDK where the errors are thrown as strings?

Reply:
Not from core SDK but from integration SDKs

Reply:
Can you fix those?

Reply:
At this moment it is not possible.